api/src/start.ts

```diff
@@ -17,7 +21,23 @@ export default async function start(): Promise<void> {
 	const port = env.PORT;
 
 	server
-		.listen(port, () => {
+		.listen(port, async () => {
+			let update = null;
+
+			try {
+				update = await checkForUpdate(pkg);
+			} catch (err) {
+				// eslint-disable-next-line no-console
+				console.log(err);
+			}
+
+			if (update) {
+				// eslint-disable-next-line no-console
+				console.log(`${logSym.info} ${chalk.yellow.bold(`New Version Available`)} (${update.latest})`);
+				// eslint-disable-next-line no-console
+				console.log('You can update by running: ' + chalk.cyan(`npx ${pkg.name}@latest [command]`));
+			}
```

Comment:
Lets use `logger.warn` instead of console.log. That also takes care of the emoji, so we don't need the `log-symbols` dependency 🙂

Reply:
@rijkvanzanten do you want me to replace the second log as well where I am telling the users how to update to the latest version? The second log is more of a message than a warning.

Reply:
They should both use logger. We could also render a single warning with a multiline string.